spec/js-api/value/calculation.d.ts.md

```diff
@@ -1,7 +1,7 @@
-# JavaScript Calculation API: Draft 2
+# JavaScript Calculation API
```

Comment:
The format of a specification document isn't the same as the format of a proposal. It doesn't need a tracking issue or changelog; it doesn't need a summary and background to explain what it's changing and why. It just needs to describe how the API should behave canonically. See the other value specifications in this directory for relevant templates.